plugins/actix-web/src/lib.rs

```diff
@@ -265,6 +280,53 @@ where
         self
     }
 
+    /// Exposes the previously built JSON specification with Swagger UI at the given path
+    ///
+    /// **NOTE:** you **MUST** call with_json_spec_at before calling this function
+    #[cfg(feature = "swagger-ui")]
+    pub fn with_swagger_ui_at(mut self, path: &str) -> Self {
```

Comment:
We should be able to return `Result<Self, Error>` here, because missing spec and rendering error can be returned here as an error instead of serving them as a response.

Reply:
Ok

Reply:
Which error type should we return?

Reply:
@wafflespeanut let me know which Error type should be returned. Otherwise, I will return a generic Error value with a message describing what happened.

Reply:
We don't seem to have any library error for the moment. Use have `thiserror` so I would recommend adding a paperclip error type, and this will be the first error we have :)